src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/expense/ExpenseController.java

```diff
@@ -0,0 +1,63 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.expense;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.expense.ExpenseService;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseResponseDTO;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("expense")
```

Comment:
Só alterar para para o plural pois representa a coleção inteira.
```suggestion
@RequestMapping("expenses")
```